### ISSUE TITLE
SUMMARY

### ISSUE THREAD

**Issue by rimmesbe** [DoKe65/colmaracademy#6]
Grade: Needs improvement

Summary: You have a really good start to this project, the desktop version looks good. You use good techniques here, but the mobile responsiveness needs a lot of work. Check out the issues I created for some tweaks here and there. I would just spend some more time on this project and finish it up. I have no doubt you are capable. https://developer.mozilla.org/en-US/docs/Mozilla/Mobile/Viewport_meta_tag

**Comment by DoKe65** [comment]
Thank’s a lot for the really fast review!

I have just a question: Since you say that it needs improvement, can I submit my work again for a last check or was this „my final chance“?

**Comment by rimmesbe** [comment]
It's a one time submission, but it shouldn't effect your completion of the
course. It's just information for you really. If you finish it and want me
to take a quick look I will.

**Comment by DoKe65** [comment]
Thank you, that’s really nice of you. I just checked my git-repository and it seems that I didn’t submit my last version of the css, which I modified only a short moment before I sent it for the review - correctly. There I wrote for the media-query: @media only screen and (max-device-width: 767px), (max-width: 767px), this works for small browser-screen to. The use of max-width only didn’t work on my iPhone, I didn’t know why and that’s why I tried with both and it worked.

Wish you a nice afternoon (I think it’s afternoon in your Timezone :-)
